action.yml

```diff
@@ -14,6 +14,9 @@ inputs:
     description: Memory limit
     required: false
     default: 512M
+  testsuite:
```

Comment:
Do we need to add this to args on runs ?

Reply:
Thanks for pointing this out. I have forgotten to add it as an argument, but I wonder how this will work if there is no testsuite parameter provided.

I'll push a new commit to add these arguments, and this can be tested on the `2-testsuite` branch.